packages/core/src/components/screens/release-notes/fetchReleaseNotes.ts

```diff
@@ -28,14 +32,16 @@ export const fetchAllReleaseNotes = async (
 ): Promise<Release[]> => {
     const scopes = getMadCommonScopes(env);
     const baseUrl = getMadCommonBaseUrl(env);
-    const authenticationResponse = await authenticateSilently(scopes);
+    const authenticationResponse = getConfig().experimental.useExpoAuthSession
+        ? await ExpoAuthSession.authenticateSilently()
```

Comment:
Probably need scopes

Reply:
it is not needed. Or do you want to add it as an unused property for now to ensure full backwards compatibility?